locales/en/public.json

```diff
@@ -197,6 +197,10 @@
     }
   },
   "DashboardLayoutToolbar": {
+    "CLEAR_ALL_CARDS":{
```

Comment:
Need to run `yarn localize` to regenerate the new locale file because in source code, you are using `'DashboardLayoutToolbar.CLEAR_ALL_CARDS'`.

If the there is file called `public.old.json` or similar, just remove it.